src/css/augmentedsteam.css

```diff
@@ -1508,6 +1508,16 @@ img.astats_icon {
   vertical-align: baseline;
 }
 
+/* Hide ds_flag and revert the burring/dimming of app images */
+.es_highlighted .ds_flag {
+  display: none;
+}
+.es_highlighted.ds_flagged img,
+.es_highlighted .ds_flagged img {
+  filter: blur(0) !important;
+  opacity: 1 !important;
```

Comment:
Why are those required?
On the search page I still see ignored items blurred, because the `opacity` is set on the `.search_result_row` itself.
I was thinking to have a permanent link to the [SteamTracking](https://github.com/SteamDatabase/SteamTracking) repo for every CSS property that we change from Steam's default values.
Because if you just look at this CSS you could ask yourself why we have to set the `opacity` to 1 here if this is actually the default value.

Reply:
`ds_flagged` class is used in several style rules to dim/blur item rows, I was testing if these two `important` rules were enough to counter all of them, apparently not...